VideoLocker/src/main/java/org/edx/mobile/view/CourseUnitWebviewFragment.java

```diff
@@ -111,8 +107,23 @@ public void onReceivedError(WebView view, int errorCode,
                 hideLoadingProgress();
                 pageIsLoaded = false;
                 ViewPagerDownloadManager.instance.done(CourseUnitWebviewFragment.this, false);
-                super.onReceivedError(view, errorCode, description, failingUrl);
             }
+
+            // Restore these annotations when we upgrade our compile SDK version to Marshmallow
```

Comment:
Hmm, so since we're ignoring the whole `.idea` settings folder in from source control, we can't share the code inspection settings at the moment either :disappointed: